shell/platform/android/android_shell_holder.cc

```diff
@@ -135,7 +139,27 @@ AndroidShellHolder::AndroidShellHolder(
         FML_LOG(ERROR) << "Failed to set Workers task runner priority";
       }
     });
+
+#if defined(__ANDROID_API__) && __ANDROID_API__ >= 28
+    if (IsNDKImageDecoderAvailable()) {
+      shell_->RegisterImageDecoder(
+          [](sk_sp<SkData> buffer) {
+            auto generator = SkImageGeneratorNDK::MakeFromEncodeNDK(buffer);
```

Comment:
I don't think we can use this API directly, because it does not dynamically look up its methods.

We are a very long time away from ever having 28 or above as a minimum API level.

Reply:
Yeah I removed this one for now.